### ISSUE TITLE
Erroneous entry points in setup.py

### ISSUE THREAD

**Issue by vmalloc** [dbcli/cli_helpers#25]
In https://github.com/dbcli/cli_helpers/blob/master/setup.py#L44, the entry points defined do not use the package name (should be `cli_helpers.tasks...`). This breaks some environments trying to load the entry points and failing on `ImportError` (mostly `pbr`-based projects)

**Comment by meeuw** [comment]
Hi @vmalloc I'm cool with that:
https://github.com/dbcli/cli_helpers/pull/26

**Comment by tsroten** [comment]
@vmalloc Hello! Thanks for reporting this issue. `tasks` is not actually inside of a `cli_helpers` package. It is a standalone module. Could you post the entire stack trace of the error you're getting?

**Comment by vmalloc** [comment]
@tsroten What do you mean by a standalone module? Where do you expect it to come from?

**Comment by vmalloc** [comment]
If I understand you correctly, I'm not sure this is correct way to achieve what you're trying to achieve. Having `tasks` in your entry points means that pkg_resources registers an ep with that name, which is susceptible to all kinds of corner cases (like somebody having the actual pypi package called `tasks` installed in his or her environment).

I lost my traceback from the incident I had yesterday, but in my case the pkg_resources inside virtualenv failed to load the ep object because it could not __import__ `tasks` (which makes sense because there's no guarantee that your code will be executed with tasks in `sys.path`)

**Comment by tsroten** [comment]
@vmalloc Yeah, that's a good point. The documentation around this kind of stuff is sparse (in my opinion). We might be trying to force `entry_points` into something it's not supposed to be.

**Comment by vmalloc** [comment]
Here is a simple reproduction of the issue:
```
Obtaining file:///Users/r/src/myproj
    Complete output from command python setup.py egg_info:
    ERROR:root:Error parsing
    Traceback (most recent call last):
      File "/Users/r/src/myproj/.eggs/pbr-3.1.1-py3.6.egg/pbr/core.py", line 111, in pbr
        attrs = util.cfg_to_args(path, dist.script_args)
      File "/Users/r/src/myproj/.eggs/pbr-3.1.1-py3.6.egg/pbr/util.py", line 267, in cfg_to_args
        wrap_commands(kwargs)
      File "/Users/r/src/myproj/.eggs/pbr-3.1.1-py3.6.egg/pbr/util.py", line 569, in wrap_commands
        cmdclass = ep.resolve()
      File "/Users/r/src/backslash/.env/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2347, in resolve
        module = __import__(self.module_name, fromlist=['__name__'], level=0)
    ModuleNotFoundError: No module named 'tasks'
    error in setup command: Error parsing /Users/r/src/myproj/setup.cfg: ModuleNotFoundError: No module named 'tasks'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /Users/r/src/myproj/
```

**Comment by vmalloc** [comment]
@tsroten is there a chance you can release a version soon without the tasks bit? It fails automated builds for people using `pgcli`...

**Comment by tsroten** [comment]
Sure! I'll get one released this week.

**Comment by vmalloc** [comment]
@tsroten awesome. Thanks!

**Comment by meeuw** [comment]
my PR isn't merged yet, can I close #26 ?

**Comment by amjith** [comment]
A new version (v1.0.0) has been released to pypi. 

Go forth and upgrade. :)
